flowbased-computation/flowbased-computation-impl/src/test/java/com/farao_community/farao/flowbased_computation/impl/FlowbasedComputationImplTest.java

```diff
@@ -113,9 +146,48 @@ public void testRun() {
         assertEquals(-0.5, getCurativePtdf(result, "N-1 FR-BE", "DE-NL - N-1 - N-1 FR-BE", "10YBE----------2"), EPSILON);
         assertEquals(0.5, getCurativePtdf(result, "N-1 FR-BE", "DE-NL - N-1 - N-1 FR-BE", "10YCB-GERMANY--8"), EPSILON);
         assertEquals(-0.5, getCurativePtdf(result, "N-1 FR-BE", "DE-NL - N-1 - N-1 FR-BE", "10YNL----------L"), EPSILON);
-
     }
 
+    /* MOCKED ANYWAY...
```

Comment:
I realized that the tests were based on a mock sensi computation. So my flowbased modifications have no effect...